### PR TITLE
Ih feature/search

### LINKED ISSUE #59: reduce welcome message size based on window size

### LINKED ISSUE #60: Search functionality

### PR DESCRIPTION
**Pull request by holladayian**
## Description
Clearly and concisely describe the feature.
This PR adds functionality to display searched items.

## Context
This was relatively simple. Mostly just html/css to get the buttons there, then filtering through recipeData with the respective search input.

## Where To Start
Scripts.js 58 and its following function are the functionality. The CSS properties start on 242.

## Affected areas of application
List all parts of the website that will be altered from this PR.
This adds two buttons to the main header. They are ```name``` and ```tag``` and add functionality to search through their respective properties.

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] Test A: DOM looks great

## Relevant Tickets
Fixes #60 
Fixes #59 

## Questions or Notes (if applicable):
This functionality only searches through all of the recipes, not favorites or recipes to cook.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,24 +12,17 @@
     <header class='main-header'>
```
Comment (Chriscastanuela) [inline src/index.html:12]: This looks nice and clean as well

File: src/scripts.js [hunk 2]
```diff
@@ -52,6 +55,22 @@ function fetchUserData() {
   .catch(err => console.log("err", err));
 }
 
+function searchRecipes(event) {
```
Comment (Chriscastanuela) [inline src/scripts.js:58]: nice search functionality

Comment (bretm9) [inline src/scripts.js:58]: Great to see search functionality implemented. This is one of the last few pieces of the project and it looks like this functionality is fully-fledged.

File: src/user.js [hunk 3]
```diff
@@ -33,6 +33,19 @@ class User {
     });
   }
 
+
+/*
```
Comment (bretm9) [inline src/user.js:37]: I'll remove these comments in my next branch.

File: src/user.js [hunk 4]
```diff
@@ -33,6 +33,19 @@ class User {
...
+  
+
+}
+
+
+
+*/
+
+
+
```
Comment (Chriscastanuela) [inline src/user.js:48]: let's come back and remove this

**Review by bretm9 — APPROVED** [review]
Thanks for removing the comments. Almost done now!

**Pull request merged**